pcdsdevices/mpodApalis.py

```diff
@@ -0,0 +1,288 @@
+import logging
+from ophyd.device import Component as Cpt
+from ophyd.device import Device
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+from pcdsdevices.interface import BaseInterface
+import time
+logger = logging.getLogger(__name__)
+
+
+class MPODApalisChannel(BaseInterface, Device):
+    """
+    MPODApalis Channel Object.
+
+    Parameters
+    ----------
+    channel_prefix : str
+        The EPICS base of the MPOD Channel, e.g. 'TMO:MPOD:01:M6:C6'.
+    name : str
+        A name to refer to the device.
+
+    """
+    voltage = Cpt(EpicsSignal, ':VoltageMeasure',
+                  write_pv=':VoltageSet', kind='normal',
+                  doc='MPOD Channel Voltage Measurement [V]')
+
+    max_voltage = Cpt(EpicsSignalRO, ':VoltageNominal', kind='normal',
+                      doc='MPOD Channel Maximum Voltage [V]')
+
+    current = Cpt(EpicsSignal, ':CurrentMeasure',
+                  write_pv=':CurrentSet', kind='normal',
+                  doc='MPOD Channel Current Measure')
+
+    max_current = Cpt(EpicsSignalRO, ':CurrentNominal', kind='normal',
+                      doc='MPOD Channel Current Maximum')
+
+    state = Cpt(EpicsSignal, ':isOn', write_pv=':Control:setOn',
+                kind='normal', string=True,
+                doc='MPOD Channel State [Off/On]')
+
+    tab_component_names = True
+    tab_whitelist = ['on', 'off',
+                     'set_voltage', 'set_current']
+
+    def on(self):
+        """Set mpod channel On."""
+        self.state.put(1)
+
+    def off(self):
+        """Set mpod channel Off."""
+        self.state.put(0)
+
+    def set_voltage(self, voltage_number):
+        """
+        Set mpod channel voltage in V.
+        Parameters
+        ----------
+        voltage_number : number
+            Voltage in V.
+        """
+        maxVoltage = self.max_voltage.get()
```

Comment:
variable names in pep8 should make this `max_voltage` https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
same below for `maxCurrent` -> `max_current`